queries/beancount/highlights.scm

```diff
@@ -0,0 +1,26 @@
+
+(date) @field
+(txn) @attribute
+
+(account) @type
+
+(amount) @number
+(incomplete_amount) @number
+(amount_tolerance) @number
+
+(currency) @property
+
+(key) @label
+(string) @string
+
+(tag) @constant
+(link) @constant
+
+(comment) @comment
+
+[
+    (BALANCE) (OPEN) (CLOSE) (COMMODITY) (PAD)
+    (EVENT) (PRICE) (NOTE) (DOCUMENT) (QUERY)
+    (CUSTOM) (PUSHTAG) (POPTAG) (PUSHMETA)
+    (POPMETA) (OPTION) (INCLUDE) (PLUGIN)
```

Comment:
Tiny tiny comment, which is absolutely not blocking: there is ways to have keywords always matched as terminals, using the `alias` function in the grammar, feel free to reach out to me on Zulip to talk about that!